src/internal/cache-client.ts

```diff
@@ -196,6 +198,107 @@ export class CacheClient {
     });
   }
 
+  public async setAddElements(
+    cacheName: string,
+    setName: string,
+    elements: string[] | Uint8Array[],
+    ttl: CollectionTtl = CollectionTtl.fromCacheTtl()
+  ): Promise<CacheSetAddElements.Response> {
+    validateCacheName(cacheName);
+    validateSetName(setName);
+    return await this.sendSetAddElements(
+      cacheName,
+      this.convert(setName),
+      this.convertArray(elements),
+      ttl.ttlMilliseconds() || this.defaultTtlSeconds * 1000,
+      ttl.refreshTtl()
+    );
+  }
+
+  private async sendSetAddElements(
+    cacheName: string,
+    setName: Uint8Array,
+    elements: Uint8Array[],
+    ttlMilliseconds: number,
+    refreshTtl: boolean
+  ): Promise<CacheSetAddElements.Response> {
+    const request = new grpcCache._SetUnionRequest({
+      set_name: setName,
+      elements: elements,
+      ttl_milliseconds: ttlMilliseconds,
+      refresh_ttl: refreshTtl,
+    });
+    const metadata = this.createMetadata(cacheName);
+    return await new Promise(resolve => {
+      this.clientWrapper.getClient().SetUnion(
+        request,
+        metadata,
+        {
+          interceptors: this.interceptors,
+        },
+        (err, resp) => {
+          if (err) {
```

Comment:
nit - lint warnings above can be avoided by simply omitting `resp` from your argument list here.  (I think this is an annoying language design choice and I think your way is more clear though so if you wanted to add the one-line eslint-ignore comment on that arg list that would be fine too.

Reply:
kk gonna tackle these as fast-follows: https://github.com/momentohq/client-sdk-javascript/issues/178